backend/src/controllers/question.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const router = express.Router({mergeParams: true})
+
+/* Get the question data for a specific question id */
+router.get('/:id', function(req, res) {
+
+    //let qid = req.params.id;
```

Comment:
presumably the idea is that this will be the actual endpoint for `api/questions`